driver/src/main/java/org/neo4j/driver/internal/logging/DelegateLogger.java

```diff
@@ -0,0 +1,425 @@
+/*
+ * Copyright (c) 2002-2017 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.neo4j.driver.internal.logging;
+
+import io.netty.util.internal.logging.InternalLogLevel;
+import io.netty.util.internal.logging.InternalLogger;
+
+import org.neo4j.driver.v1.Logger;
+
+import static java.lang.String.format;
+
+
+public class DelegateLogger implements InternalLogger
```

Comment:
How about calling this smth like `NettyLogger`?
I think we could also extend `io.netty.util.internal.logging.AbstractInternalLogger` to reduce amount of code.